src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,26 @@
+package racingcar.domain
+
+data class Cars(private val cars: List<Car>) {
+    fun move(condition: () -> Int): List<Car> {
+        val of = mutableListOf<Car>()
+
+        cars.forEach {
+            of.add(it.move(condition()))
+        }
+        return of
+    }
```

Comment:
```suggestion
    fun move(condition: () -> Int): List<Car> {
        return cars.map {
            it.move(condition())
        }
    }
```

`map`을 사용하면 로직이 간결해지지 않을까요?

Reply:
@Flamme1004K 반영했습니다!